### PR TITLE
cloud: update tikv/tiflash node storage

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update doc for [DM-6179](https://jira.tidbcloud.com/browse/DM-6179).

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### CODE REVIEW

File: tidb-cloud/size-your-cluster.md [hunk 1]
```diff
@@ -88,9 +88,12 @@ Minimum number of TiKV nodes: `ceil(2048 ÷ 0.8 ÷ 1024) × 3 = 9`
 
 ### TiKV node storage
```
Comment (SunRunAway) [inline tidb-cloud/size-your-cluster.md:89]: minimum storage size of TiFlash is set to 200 GiB, too

**Review by qiancai — APPROVED** [review]
LGTM

**Review by SunRunAway — APPROVED** [review]

**Pull request merged**